app/src/components/Common/Link/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { Link as MuiLink, LinkProps } from "@mui/material";
+import { useCallback } from "react";
+
+type LinkComponentType = {
+	href?: string;
+	onClick?: () => void;
+	children?: React.ReactNode;
+} & LinkProps;
+
+export default function LinkComponent({
+	children,
+	onClick,
+	href,
+	...props
+}: LinkComponentType) {
+	const onEvent = useCallback(
```

Comment:
op-pa
nashla koca na kamenb
for what purposes do we use useCallback? What are the differences between useCallback and useMemo? When should useCallback and useMemo be used and when should they be ignored? I think we should discuss this in telegramm

Reply:
oopsii, let's talk abou it